tests/trnsysGUI/storageTank/testConfigureStorageDialog.py

```diff
@@ -15,26 +17,28 @@
 class TestConfigureStorageDialog:
     storageTank = None
 
-    def triggerStorageDialog(self, qtbot):
+    @pytest.fixture()
+    def triggerStorageDialog(
+        self, qtbot, monkeypatch
+    ) -> ConfigureStorageDialog:
+        monkeypatch.setattr(QMessageBox, "exec", lambda self: QMessageBox.Ok)
```

Comment:
Can we maybe patch `QMessageBox.information` or (`warning`, or whichever is appropriate, directly?). Might require small changes in productive code.

Reply:
Thanks @zuckerruebe.

Not sure if that would work.
The main issue is that pytest grinds to a halt until the dialog is closed.
Thus, the problem to solve is: 
**_ensuring that exec() is passed cleanly during testing._**

Does the current patch seem better with this information, or would you still prefer patching something else to reach the same goal?

Reply:
Additionally, I think a bigger overhaul of this is in order.
Two architectural choices would be helpful here.
1: The current state should be saved in memory, so that the entire state can be reproduced at the end of the export.
2: MVP or MVVM, so we do not have to deal with the GUI part at all in these tests.

This would not be part of this PR.

Reply:
I actually don't know why I get e-mails regarding this pull request. But because I do, I felt I'd chirp in. But I don't have a strong opinion nor a lot of context.

If it's possible to use, e.g., `QMessageBox.information`, then, if you monkeypatch that one `.exec` will never get called because it's called within `information`.